ui/src/modules/onboarding/iframe.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+export function showIframe(url) {
+  const wrapper = document.createElement('div');
+  const shadowRoot = wrapper.attachShadow({ mode: 'closed' });
+  const template = document.createElement('template');
+
+  template.innerHTML = `
+    <iframe src="${url}" frameborder="0"></iframe>
+    <style>
+      :host {
+        all: initial !important;
```

Comment:
@smalluban please take a look on this. some pages have styles that are as broad as `div`. Eg. example.com